features/shop/checkout/being_able_to_modify_remaining_steps.feature

```diff
@@ -15,59 +15,59 @@ Feature: Changing checkout steps
 
     @no-api @ui @javascript
     Scenario: Changing address of my order
-        Given I had product "PHP T-Shirt" in the cart
+        When I add product "PHP T-Shirt" to the cart
```

Comment:
This one should Given, the same for below scenarios and the js tag probably could be removed